onnx/test/test_backend_reference.py

```diff
@@ -161,7 +161,9 @@ def run_node(cls, node, inputs, device=None, outputs_info=None, **kwargs):
 
 # The following tests fail due to a shape mismatch.
 backend_test.exclude(
-    "(test_center_crop_pad_crop_axes_hwc_expanded|test_lppool_2d_dilations)"
+    "(test_center_crop_pad_crop_axes_hwc_expanded"
+    "|test_lppool_2d_dilations"
+    "|test_averagepool_2d_dilations)"
```

Comment:
The reference evaluation will be fixed in another PR?

Reply:
yes https://github.com/onnx/onnx/issues/5161
this PR is for 1.14 release. Ort release depend on this PR so need to merge and cherry-pick by release sooner the better. Fix of reference implementation can come after.